### ISSUE TITLE
Improve rendering of non-wrapped elements

### ISSUE THREAD

**Issue by ekuleshov** [daohoangson/flutter_widget_from_html#554]
## Use case

When rendering tables it seems like the `<nobr>` tag is ignored, so as the `nowrap` CSS declaration. The following html fragments are rendered with a wrapped text

`<td><nobr>Kuleshov, Eugene</nobr></td>`

and

`<td class="score_cell">99.0000&nbsp;%</td>`

where CSS is declared like this in the same html:

`.score_cell {text-align:right; white-space:nowrap;}`

## Proposal

Please add handling of non-wrapped text or allow to provide some widget factory to support that.

**Comment by daohoangson** [comment]
Both the `NOBR` tag and the `nowrap` declaration are unsupported for now. Using `&nbsp;` should work though. Your scroll cell is still being wrapped into the next line?

**Comment by ekuleshov** [comment]
Yes, text in cells is being wrapped. I realize it is not supported, hence this request.

Which widget is being used to render text inside the table cells and the whole table grid? I was wondering if something like `Text(softWrap:false)` could be used. In my case horizontal scrolling is acceptable, if needed.

Unfortunately I'm getting external htmls and would have to re-process them in order to replace all css and nobr tags, which is not desirable.

**Comment by daohoangson** [comment]
This package uses `RichText` directly, it does support `softWrap` like the `Text` widget. You can use a custom `WidgetFactory` and override the `buildText` method to return a rich text without soft wrap I think. 

https://github.com/daohoangson/flutter_widget_from_html/blob/0532428db0ba2003e36211faaca1199313675f8e/packages/core/lib/src/core_widget_factory.dart#L313
